Library/Formula/beanstalk.rb

```diff
@@ -12,6 +10,7 @@ class Beanstalk < Formula
   end
 
   def install
+    mkdir_p var/"log"
```

Comment:
I wasn't sure either way. It seems to be regularly placed in ` install `, and only placed in `postinstall` when `postinstall` exists for some other reason. Wasn't sure if that was a deliberate maintainer choice over time, or.